lib/view/page/splash_page.dart

```diff
@@ -23,6 +25,10 @@ class _SplashPageState extends State<SplashPage> {
       // ensure the onComplete callback cannot be called in the same frame
       widget.onComplete();
     });
+    await Firebase.initializeApp();
+    FirebaseCrashlytics.instance.setCrashlyticsCollectionEnabled(true);
+    /// Pass all uncaught errors from the framework to Crashlytics.
+    FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterError;
```

Comment:
Firebase should be initialized before the onComplete callback. Should be placed under line 23.